crypto/bn/bn_kron.c

```diff
@@ -11,7 +11,7 @@
 #include "bn_lcl.h"
 
 /* least significant word */
-#define BN_lsw(n) (((n)->top == 0) ? (BN_ULONG) 0 : (n)->d[0])
+#define BN_lsw(n) ((n)->top == 0 ? (BN_ULONG) 0 : (n)->d[0])
```

Comment:
Missed opportunity of removal of excessive space. I mean `(BN_ULONG)0` should be without space.

Reply:
I've done and undone this, and haven't redone the space removal the second time. It's an iterative process, I prefer multiple smaller passes with clear changes, than grabbing all opportunity at once. It makes reviewing changes easier, though with more "churn" (aka number of no-value updates). It also takes quite a bit of time to get all the rules and feel what core members want to see in each particular case.